### PR TITLE
Cleanup tests and C extensions of unused Python version

### PR DESCRIPTION
**Pull request by starnight**
Cleanup tests and C extensions of unused Python version to reduce flatpak size.

### CODE REVIEW

**Comment by starnight** [comment]
I think this is good to be in upstream [kolibri-installer-gnome](https://github.com/learningequality/kolibri-installer-gnome).  But, I do not have permission to push a branch to it.

File: build-aux/flatpak/org.endlessos.Key.Devel.json [hunk 1]
```diff
@@ -70,6 +70,15 @@
...
             ]
+        },
+        {
+            "name" : "python3-klobri-cleanup",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "find ${KOLIBRI_MODULE_PATH}/dist/cext -type d -name 'cp*' -not -path ${KOLIBRI_MODULE_PATH}/dist/cext/cp310 -exec rm -rf '{}' '+'",
+                "rm -rf ${KOLIBRI_MODULE_PATH}/dist/cext/*/Windows",
+                "find ${KOLIBRI_MODULE_PATH} -type d -name 'test*' -not -path ${KOLIBRI_MODULE_PATH}/dist/django/test -exec rm -rf '{}' '+'"
+            ]
```
Comment (GeorgesStavracas) [inline build-aux/flatpak/org.endlessos.Key.Devel.json:81]: Wouldn't it be better to move these commands to a `cleanup-commands` array within each module? Usually the last module is restricted to the app of the manifest.

Comment (starnight) [reply build-aux/flatpak/org.endlessos.Key.Devel.json:81]: Good suggestion!  Moving it to `cleanup-commands` makes sense.
However, according to [Flatpak's manifests document](https://docs.flatpak.org/en/latest/manifests.html#cleanup), modules only support `cleanup`, no `cleanup-commands`.  And, we have to skip paths, like `${KOLIBRI_MODULE_PATH}/dist/cext/cp310` and `${KOLIBRI_MODULE_PATH}/dist/django/test` from the remove list.

So, simply moving the cleanup commands to `org.endlessos.Key.Devel.json`'s **cleanup-commands** section is a better way.  Then, we do not bother **modules** of `org.endlessos.Key.Devel.json`.

**Review by GeorgesStavracas — APPROVED** [review]

**Comment by starnight** [comment]
Checked with new built flatpak.  Those paths have been removed correctly.

**Pull request merged**